content/techniques/sessions.md

```diff
@@ -35,6 +35,29 @@ Likewise, enabling the `saveUninitialized` option Forces a session that is "unin
 
 You can pass several other options to the `session` middleware, read more about them in the [API documentation](https://github.com/expressjs/session#options).
 
+In case we need to save our session in a storage such as `Redis`, we should add the `store` option in our session, as follows:
+
+```typescript
+import * as RedisStore from 'connect-redis';
+import * as Redis from 'redis';
+
+const redisClient = Redis.createClient({
+  url: 'redis://:@127.0.0.1:6379',
+  legacyMode: true,
+});
+redisClient.connect().catch(console.error);
```

Comment:
Not sure what I think about this local catch/connect here 😄 Although this example seems to be quite useful, it doesn't seem to follow the best practices 🤔

Reply:
I wanted to pick up this PR and write it a little better actually I don't like it either 😅, but I'll see what I can do 😄